draft-tuexen-opsawg-pcapng.xml

```diff
@@ -1371,6 +1400,39 @@ Section Header
 
             <t>Example: '0'.</t>
 
+            <t hangText="epb_packetid:"><vspace blankLines="0"/>The
+            epb_packetid option is a 64-bit unsigned integer that uniquely
+            identifies the packet. If the same packet is seen by multiple
+            interfaces and there is a way for the capture application to
+            correlated them, the same epb_packetid value must be used. An
+            example could be where an XDP program on ingress captures a packet,
+            which gets modified and then exits the XDP program. In this case,
+            two packets are in the capture file, which are not identical but
+            the epb_packetid can be used correlate them.</t>
+
+            <t>Example: '0'.</t>
+
+            <t hangText="epb_queue:"><vspace blankLines="0"/>The epb_queue
+            option is a 32-bit unsigned integer that identifies on which queue
+            of the interface the specific packet was received.</t>
+
+            <t>Example: '0'.</t>
+
+            <t hangText="epb_verdict:"><vspace blankLines="0"/>The epb_verdict
+            option stores a verdict of the packet. The verdict indicates what
+            would be done with the packet after processing it. For example, a
+            firewall implemented with eBPF could drop the packet. This verdict
+            can be set by various components, i.e. Hardware, Linux's eBPF TC
+            or XDP framework, etc. etc. The first octet specifies the verdict
+            type, while the following octets contain the actual verdict data,
+            whose size depends on the verdict type, and hence from the value
+            in the first octet. The verdict type can be: Hardware (type
+            octet = 0, size = XXX), Linux_eBPF_TC (type octet = 1, size = 8
```

Comment:
These should probably point to a source indicating what the values for Linux eBPF TC and Linux eBPF XDP values signify.

Reply:
What about:

The verdict type can be: Hardware (type
            octet = 0, size = variable), Linux_eBPF_TC (type octet = 1, size = 8
            (64-bit unsigned integer), value = TC_ACT_* as defined in the Linux
            <eref target="https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/include/uapi/linux/pkt_cls.h">pck_cls.h</eref>
            include), Linux_eBPF_XDP (type octet = 2, size = 8 (64-bit unsigned
            integer), value = xdp_action as defined in the Linux
            <eref target="https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/include/uapi/linux/bpf.h">pbf.h</eref>
            include).</t>

Reply:
So where can these "pck_cls.h" and "pbf.h" files be found?  They can't be found in any obvious place in the Linux 5.6.7 kernel source, at least not if the source tarball is downloaded and unpacked and a "find" command is run in the top-level directory.

Reply:
Nice... The Github editor removed the ref links I put in... 

```
The verdict type can be: Hardware (type
            octet = 0, size = variable), Linux_eBPF_TC (type octet = 1, size = 8
            (64-bit unsigned integer), value = TC_ACT_* as defined in the Linux
            <eref target="https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/include/uapi/linux/pkt_cls.h">pck_cls.h</eref>
            include), Linux_eBPF_XDP (type octet = 2, size = 8 (64-bit unsigned
            integer), value = xdp_action as defined in the Linux
            <eref target="https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/include/uapi/linux/bpf.h">pbf.h</eref>
            include).</t>
```